internal/services/serviceconnector/service_connector_app_service_resource.go

```diff
@@ -222,6 +292,18 @@ func (r AppServiceConnectorResource) Read() sdk.ResourceFunc {
 					state.SecretStore = flattenSecretStore(*props.SecretStore)
 				}
 
+				if props.Scope != nil {
+					state.Scope = pointer.From(props.Scope)
+				}
```

Comment:
Just a note, the `pointer.From()` function handles the `nil` for you, and because the nil returns the empty value for the base type it's safe to just use:
```suggestion
				state.Scope = pointer.From(props.Scope)
```